src/components/Property/Types/VectorProperty/ViewOptions/ColorVector.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, RGBA } from '@mantine/core';
+
+import { ColorPicker } from '@/components/ColorPicker/ColorPicker';
+import { VectorPropertyProps } from '@/components/Property/Types/VectorProperty/VectorProperty';
+
+import { ValueList } from './DefaultValueList';
+
+export function ColorVector(props: VectorPropertyProps) {
+  const { disabled, setPropertyValue, value, isInt } = props;
+
+  if (value.length < 3 || value.length > 4 || isInt) {
```

Comment:
```suggestion
  if (value.length !== 3 || value.length !== 4 || isInt) {
```

Reply:
It would have to look like this: 
```
 if ((value.length !== 3 && value.length !== 4) || isInt) {
```
but sure